### ISSUE TITLE
Device sensor identification / database?

### ISSUE THREAD

**Issue by fitzy89** [vitalidze/traccar-web#285]
Hello

Do you happen to know if there is a database anywhere of various sensors and values for supported devices?

I have a Ruptela FM-Eco3 tracker and I've done a bit of testing to identify which sensors shown in Traccar are for which function of the device. It may not be accurate but from my experience so far I think I've got it right. This tracker has 4 digital inputs, 2 analogue inputs and 2 analogue outputs.

I am now trying to work out the functions of io27 and io65.

| Traccar sensor | Ruptela FM-Eco3 function | Notes |
| --- | --- | --- |
| io2 | DIN1 | Digital input 1 |
| io3 | DIN2 | Digital input 2 |
| io4 | DIN3 | Digital input 3 |
| io5 | DIN4 | Digital input 4. By default, is set to ignition status (off/on = 0/1) |
| io27 |  | Unidentified function, my values vary between 11-17 while driving |
| io65 |  | Unidentified function, possibly power/battery supply voltage as  values fluctuate around the same level for the entre journey. Values appear as a 8 digit number, possibly should have a decimal? |
| sat | Number of satellites |  |
| ip | IP Address |  |

**Comment by vitalidze** [comment]
No, I don't know about such database. I believe the best place to find information about all these sensors is the device protocol documentation. You can find many documentations here: https://www.traccar.org/protocols/

**Comment by vitalidze** [comment]
I have just remembered [this one](http://wialon.com/en/hardware/Device?HwDeviceSearch%5Bmanufacturer_id%5D=13&HwDeviceSearch%5Bcategory_id%5D=&HwDeviceSearch%5Bname%5D=&page=2) - there are some sensor descriptions. However, the sensor names may not match with traccar's sensor names.

**Comment by fitzy89** [comment]
Thanks again @vitalidze I've looked at both the links you mention and although they are very useful sadly they don't have the information I'm looking for. I've been searching for a while today online but I can't find much about it really even in the manual it doesn't mention the functions of the reported sensors but hopefully I'll be able to work it out. I'll leave this post up and maybe it might help someone else. Thanks for your time once again.
